src/gui/explorer.go

```diff
@@ -42,6 +31,8 @@ type CoinSupply struct {
 	TotalSupply string `json:"total_supply"`
 	// MaxSupply is the maximum number of coins to be distributed ever
 	MaxSupply string `json:"max_supply"`
+	// CoinHourSupply is the total number of coinhours in the distribution addresses
+	CoinHourSupply string `json:"coinhour_supply"`
```

Comment:
Can we do `current_coinhour_supply` (exclude all distribution addresses) and `total_coinhour_supply` (same as `coinhour_supply` is here) to match `current_supply` and `total_supply`?